web/src/muti_language.svg

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" standalone="no"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg t="1626179895478" class="icon" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" p-id="1276" xmlns:xlink="http://www.w3.org/1999/xlink" width="200" height="200"><defs><style type="text/css">html, * { user-select: text !important; }
```

Comment:
Static resources like SVG should be put to static: https://github.com/casbin/static/tree/master/img , then refer to to with URL

Reply:
@turbodog03 merged: https://github.com/casbin/static/pull/22